commit 8f5d9e68c90dd6b01afdfda6d9926c6ea4931c00
Author: Johannes Berg <johannes.berg@intel.com>
Date:   Fri Jul 15 15:04:41 2022 +0200

    wifi: mac80211: remove stray printk
    
    Unfortunately, a printk snuck into a previous patch,
    remove it.
    
    Fixes: 81151ce462e5 ("wifi: mac80211: support MLO authentication/association with one link")
    Signed-off-by: Johannes Berg <johannes.berg@intel.com>

diff --git a/net/wireless/mlme.c b/net/wireless/mlme.c
index defce1530115..003c57504583 100644
--- a/net/wireless/mlme.c
+++ b/net/wireless/mlme.c
@@ -50,7 +50,6 @@ void cfg80211_rx_assoc_resp(struct net_device *dev,
 		/* need to have local link addresses for MLO connections */
 		WARN_ON(cr.ap_mld_addr && !cr.links[link_id].addr);
 
-		printk(KERN_CRIT "BSS pointer 0x%lx\n", (unsigned long)cr.links[link_id].bss);
 		BUG_ON(!cr.links[link_id].bss->channel);
 
 		if (cr.links[link_id].bss->channel->band == NL80211_BAND_S1GHZ) {